configs/mender_convert_config

```diff
@@ -32,9 +32,21 @@ MENDER_ARTIFACT_COMPRESSION="gzip"
 # this.
 MENDER_ENABLE_SYSTEMD=y
 
+
+# Custom mkfs options for creating the rootfs partition
+MENDER_ROOT_PART_MKFS_OPTS=""
+
+# Set the fstab options for mounting the root partition
+MENDER_ROOT_PART_FSTAB_OPTS="defaults"
+# Set to 0 to disable fsck; needed for read-only rootfs support
+MENDER_ROOT_PART_FS_PASSNO="1"
```

Comment:
This can be one configuration option, e.g:

```
MENDER_ROOT_PART_FSTAB_OPTS = "defaults 1 1"
```

Reply:
Yes, but in Yocto the same variable is used only for the first part, so it's probably better to keep them split for consistency. The dump and pass number aren't configurable in Yocto ATM.